ckanext/honduras/templates/home/snippets/featured_organization.html

```diff
@@ -0,0 +1,9 @@
+{% set organizations = h.get_featured_organizations() %}
```

Comment:
You can pass a parameter with the number of preferred orgs / groups :

```
{% set organizations = h.get_featured_organizations(3) %}
```
By default it will get the 3 most recent ones, but if we really want to choose which ones are appearing we can use the `ckan.featured_groups` and `ckan.featured_orgs` config options, eg:

```
ckan.featured_groups = economy environment birds
```